core/load_balancer_rule.go

```diff
@@ -74,6 +74,8 @@ type LoadBalancerRuleArguments struct {
 	ListenPort        int                       `json:"listen_port,omitempty"`
 	Protocol          Protocol                  `json:"protocol,omitempty"`
 	ProxyProtocol     *bool                     `json:"proxy_protocol,omitempty"`
+	BackendSSL        *bool                     `json:"backend_ssl,omitempty"`
+	PassthroughSSL    *bool                     `json:"passthrough_ssl,omitempty"`
```

Comment:
These fields should be added to the `TestLoadBalancerRuleArguments_JSONMarshalling` test as well, so we can verify they are marshaled and unmarshaled correctly.